Orm/Xtensive.Orm/Orm/Domain.cs

```diff
@@ -119,8 +119,9 @@ public static Domain Demand()
 
     internal KeyGeneratorRegistry KeyGenerators { get; private set; }
 
-    internal ConcurrentDictionary<TypeInfo, IReadOnlyList<PrefetchFieldDescriptor>> PrefetchFieldDescriptorCache { get; private set; }
-    
+    internal ConcurrentDictionary<TypeInfo, IReadOnlyList<PrefetchFieldDescriptor>> PrefetchFieldDescriptorCache { get; } =
+       new ConcurrentDictionary<TypeInfo, IReadOnlyList<PrefetchFieldDescriptor>>();
+
```

Comment:
Why don't you just remove ```private set``` part?

Reply:
in-place initialization improves readability

Reply:
In general yes but what does this another mentioning of property type contributes to readability?

Reply:
fewer lines of code.
Are you suggesting to put it back into constructor?

Reply:
reverted it